src/utils/Enums.ts

```diff
@@ -0,0 +1,17 @@
+export enum TIMES {
+  NOTIFICATION_TIME = 4,
+  TERMINATION_TIME = 5
+}
```

Comment:
Wouldn't it be better to keep this configuration in the config.yml file?

Reply:
50/50... getting configs is more difficult, and no more flexible than enums. Will change if there's a strong preference, but otherwise not